plugins/colorhug/fu-colorhug-device.c

```diff
@@ -160,6 +160,7 @@ fu_colorhug_device_open (FuUsbDevice *device, GError **error)
 	fu_device_add_guid (FU_DEVICE (device), ch_device_get_guid (usb_device));
 	fu_device_add_icon (FU_DEVICE (device), "colorimeter-colorhug");
 	fu_device_add_flag (FU_DEVICE (device), FWUPD_DEVICE_FLAG_UPDATABLE);
+	fu_device_set_remove_delay (FU_DEVICE (device), FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE);
```

Comment:
so previously you were looking for a 10000 delay and now it's 5000.  Did I see that right?  Shouldn't this be `2 * FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE`

Reply:
I think 10 seconds is a very long timeout indeed for re-enumeration -- `FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE` is supposed to be a "good default" that plugins can ignore if they know the device is going to take longer, e.g. the Jabra detach.